woocommerce/woocommerce-ios
# Store creation: Remove outdated feature flags and enable store creation by default

⚠️ Please make sure that #10360 is reviewed and merged first.
Closes: #10327 


## Description

This PR concludes the code cleanup for store creation. Since we decided to remove the web view solution for store creation, the feature flag `storeCreationM2WithInAppPurchasesEnabled` should be removed along with `storeCreationM2`. Store creation is enabled by default with a single code path for free trial.

## Testing instructions

- Build, run, and log out of the app if needed.
- On the prologue screen, notice that both CTA is available as before: Log In and Get Started.
- Select Get Started and tap Log In.
- Enter an email address that is not associated to any WPCom account.
- See the error screen about incorrect email address > the CTA to Create an account should be available.

<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/720b900a-b7b2-49e4-8172-d117011de776" width=320 />

- Navigate back and enter the correct email address to finish login.
- After successfully logging in, switch to the Menu tab and open the site picker.
- Select Add a store > Create a new store.
- The store creation flow should work as before.

## Screenshots

https://github.com/woocommerce/woocommerce-ios/assets/5533851/f4d950d3-b61c-4c16-abb1-7396c891680a

---
